artman/cli/main.py

```diff
@@ -227,6 +227,14 @@ def parse_args(*args):
         action='store_true',
         help='[Optional] When specified, artman will skip the remote '
         'publishing step.', )
+    parser_publish.add_argument(
+        '--local-repo-dir',
+        type=str,
+        default=None,
+        help='[Optional] When specified, artman will skip the lookup local '
+        'publishing configuration in ~/.artman.config.yaml, nor clone a new '
+        'github repo. Instead, it will use the specified directory to stage '
+        'generated result. This is only working when under dry run mode.', )
```

Comment:
s/This is only working when under /This only works under/

Reply:
Done.